src/stories/Library/event-header/EventHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from "react";
+import { Tag } from "../tag/Tag";
+import { Button } from "../Buttons/button/Button";
+
+type EventHeaderProps = {
+  title: string;
+  date: string;
+};
+
+const EventHeader: FC<EventHeaderProps> = ({ title, date }) => {
+  return (
+    <header className="event-header">
+      <section className="event-header__content">
+        <div className="event-header__extra-padding">
+          <Tag hasBackground className="event-header__tag">
+            UDSTILLING
+          </Tag>
+          <time className="event-header__date">{date}</time>
+          <h1 className="event-header__title">{title}</h1>
+        </div>
+        <Button
+          classNames="event-header__button"
+          label="Køb billet"
+          buttonType="none"
+          disabled={false}
+          collapsible={false}
+          size="large"
+          variant="filled"
+        />
```

Comment:
Please change this to a link.

While the element visually looks like a button in HTML semantics it is actually a link to an external page.